templates/apiary-bucket-policy.json

```diff
@@ -150,6 +150,35 @@
             ]
         },
 %{endif}
+%{if common_producer_iamroles != ""}
+        {
+            "Sid": "General read-write iamrole permissions",
+            "Effect": "Allow",
+            "Principal": "*",
+            "Action": [
+                "s3:GetBucketLocation",
+                "s3:GetObject",
+                "s3:GetObjectAcl",
+                "s3:GetBucketAcl",
+                "s3:ListBucket",
+                "s3:PutObject",
+                "s3:PutObjectAcl",
+                "s3:DeleteObject",
+                "s3:GetBucketVersioning",
```

Comment:
pls add tagging permissions in case:
`PutObjectTagging`
`GetObjectTagging`

Reply:
Added the tagging permissions.